load/type.go

```diff
@@ -89,6 +89,12 @@ func (t *Naked) LoadFields(levels int) error {
 		return nil
 	}
 	for i := 0; i < t.st.NumFields(); i++ {
+		m := parseTags(t.st.Tag(i))
```

Comment:
I'm a bit uncomfortable with the fact that for each field we parse the tags twice, once in line 92, and once inside `newField` in line 98.
Maybe add an field to the Field type, 'Ignored' that if it true we continue in this loop?

Reply:
@posener  we can try to parse the tags once and pass them down as argument. there is lots of logic before the call to parsing tags, which are supposed to be skipped since we know the field has to be skipped.

https://github.com/posener/orm/blob/6123c3f02e7a154e5ce9190a2213c738e1a8fd46/load/field.go#L54-L79

Reply:
Good point,
Maybe move the `err = f.parseTags(parent.st.Tag(i))` to https://github.com/posener/orm/blob/6123c3f02e7a154e5ce9190a2213c738e1a8fd46/load/field.go#L60
And if there is a dash in the tags, `return nil, nil`?

Reply:
that wont work since the call is attached to a struct as a receiver. Allocating a new Field only not to use it is unnecessary allocation.

Please check the changes I added in a second commit and tell me what you think.